regulations/tasks.py

```diff
@@ -39,7 +46,8 @@ def submit_comment(body):
                 'X-Api-Key': settings.REGS_GOV_API_KEY,
             }
         )
-        response.raise_for_status()
+        if response.status_code != requests.codes.created:
```

Comment:
Are there other things that could go wrong that would merit a retry? What if fetching a file from s3 fails?